src/pages/settings/Profile/PersonalDetails/LegalNamePage.js

```diff
@@ -36,92 +37,73 @@ const defaultProps = {
     },
 };
 
-class LegalNamePage extends Component {
-    constructor(props) {
-        super(props);
-
-        this.validate = this.validate.bind(this);
-        this.updateLegalName = this.updateLegalName.bind(this);
-    }
+const updateLegalName = (values) => {
+    PersonalDetails.updateLegalName(
+        values.legalFirstName.trim(),
+        values.legalLastName.trim(),
+    );
+};
 
```

Comment:
Why we brought this outside?

Reply:
Following the conclusion of this [discusssion](https://expensify.slack.com/archives/C01GTK53T8Q/p1680096510744619)